common/src/cellular_pkthandler.c

```diff
@@ -107,6 +107,8 @@ static CellularPktStatus_t _convertAndQueueRespPacket( CellularContext_t * pCont
         }
 
         /* Notify calling thread, Not blocking immediately comes back if the queue is full. */
+        /* This is platform dependent api. */
+        /* coverity[misra_c_2012_directive_4_6_violation] */
         if( xQueueSend( pContext->pktRespQueue, ( void * ) &pktStatus, ( TickType_t ) 0 ) != pdPASS )
```

Comment:
Sorry to mention here. Queue functions are support in platform. Suggest we create another PR to move to cellular_platform.h.

Reply:
Does that mean you want to have a default cellular_platform.h file? 
In our case, we will have two cellular_platform.h file for our application.
1. CBMC
2. unit-test
And Coverity will take use of the setting from unit-test (referring to other released library).
I think we don't have to provide this file because the platform functions are variant, how do you think?

Reply:
Cellular_platform.h is used to separate RTOS dependency with this library.
To keep consistency, I suggest we use cellular_platform.h to define these dependency.
The naming rule should be consistent with [document](https://www.freertos.org/Documentation/api-ref/cellular/cellular_platform.html).
During the discussion, a cellular_platform.h template can help developer to integrate this library.
It is not related to MISRA PR. We can add this item in naming PR.